### ISSUE TITLE
Plugin can ONLY find .env file if dotenvOptions.path is set with an absolute path

### ISSUE THREAD

**Issue by PsiRadish** [mrmckeb/typescript-plugin-css-modules#56]
**Describe the bug**
If `dotenvOptions.path` is not set then `dotenv` will proceed with its default path of
```js
path.resolve(process.cwd(), '.env')
```
and since `process.cwd()` is the VS Code installation directory, it will not find anything. Setting `dotenvOptions.path` to a relative path also fails, for likely the same reason.

**To Reproduce**

1. **In some React project**
   Add a `.env` file at the project root containing
   ```
   HELLO=WORLD
   ```
1. **In `typescript-plugin-css-modules`**
   Add the following after [the `dotenv.config` call in index.ts](https://github.com/mrmckeb/typescript-plugin-css-modules/blob/v2.0.0/src/index.ts#L36) and rebuild:
   ```ts
   logger.log(`process.env.HELLO after dotenv.config = ${process.env.HELLO}`);
   ```
1. Put the above modified version of `typescript-plugin-css-modules` in the React project's `node_modules`
1. Add `typescript-plugin-css-modules` to `compilerOptions.plugins` in the React project's `tsconfig.json` with no additional options
1. Reload Window in VS Code for the React project, then view a ts/tsx file and open the TS Server log
1. `tsserver.log` contains:
   ```
   [typescript-plugin-css-modules] process.env.HELLO after dotenv.config = undefined
   ```
   ***Loading from default location with `dotenvOptions.path` unset = FAILURE***
1. Add the following options for `typescript-plugin-css-modules` to `tsconfig.json`:
   ```json
   "options": {
     "dotenvOptions": {
       "path": "./.env"
     }
   }
   ```
1. Reload Window in VS Code, view the ts/tsx file again, and open the TS Server log
1. `tsserver.log` contains:
   ```
   [typescript-plugin-css-modules] process.env.HELLO after dotenv.config = undefined
   ```
   ***Loading from relative path in `dotenvOptions.path` = FAILURE***
1. Set the following options for `typescript-plugin-css-modules` in `tsconfig.json`:
   ```json
   "options": {
     "dotenvOptions": {
       "path": "C:/Full/Absolute/Path/to/Project/.env"
     }
   }
   ```
1. Reload Window in VS Code, view the ts/tsx file again, and open the TS Server log
1. `tsserver.log` shows:
   ```
   [typescript-plugin-css-modules] process.env.HELLO after dotenv.config = WORLD
   ```
   ***Loading from absolute path in `dotenvOptions.path` = ONLY SUCCESS***

**Expected behavior**
1. Plugin can direct `dotenv` to load a `.env` file located at the root of the project without `dotenvOptions.path` having to be set
1. Plugin can direct `dotenv` to load a file from a relative path in `dotenvOptions.path`

**Comment by mrmckeb** [comment]
Hi @PsiRadish, I think we can fix this by adding a default config (path = project path). Does that make sense to you?

**Comment by PsiRadish** [comment]
Absolutely.

Though, just to be clear, do you mean doing something like this in `create`?
```ts
if (!dotenvPath) { // no custom path; fall back on <project root>/.env
  dotenvPath = path.resolve(info.project.getCurrentDirectory(), '.env');
}
else if (!path.isAbsolute(dotenvPath)) { // relative custom path; manually resolve relative to project root
  dotenvPath = path.resolve(info.project.getCurrentDirectory(), dotenvPath);
}
```
Or is there some other way of doing it?

**Comment by mrmckeb** [comment]
I'd suggest instead, just creating a default object

https://github.com/mrmckeb/typescript-plugin-css-modules/blob/develop/src/index.ts#L36

```
dotenv.config({
  path: path.resolve(info.project.getCurrentDirectory(), '.env'),
  ...options.dotenvOptions,
});
```

And in other cases, the user can replace it with their options... sound OK?

**Comment by mrmckeb** [comment]
If so, I can do today/tomorrow ;)

**Comment by mrmckeb** [comment]
How does this look @PsiRadish? If you think it looks OK, I'll merge and release today!

**Comment by mrmckeb** [comment]
Sorry @PsiRadish - https://github.com/mrmckeb/typescript-plugin-css-modules/pull/58

**Comment by PsiRadish** [comment]
That looks spiffy.

**Comment by PsiRadish** [comment]
😬 😅
So the `dotenvOptions` object the plugin constructs [here](https://github.com/mrmckeb/typescript-plugin-css-modules/blob/v2.0.1/src/index.ts#L36-L42) looks great.
Unfortunately it never gets used, as `dotenv.config` is [still called with the original `options.dotenvOptions` afterward](https://github.com/mrmckeb/typescript-plugin-css-modules/blob/v2.0.1/src/index.ts#L43).

Once I fixed that, though, everything else seemed to work.

**Comment by mrmckeb** [comment]
Thanks @PsiRadish, this is hopefully now resolved (#59). Please see [v2.0.2](https://github.com/mrmckeb/typescript-plugin-css-modules/releases/tag/v2.0.2).

Thanks again!
